src/lib/utils.ts

```diff
@@ -133,53 +134,53 @@ export function buildClientState(
       hash: HashOp.SHA256,
       prehashValue: HashOp.SHA256,
       prehashKey: HashOp.NO_HASH,
-      length: LengthOp.VAR_PROTO,
+      length: LengthOp.VAR_PROTO
     },
     innerSpec: {
       childOrder: [0, 1],
       minPrefixLength: 4,
       maxPrefixLength: 12,
       childSize: 33,
-      hash: HashOp.SHA256,
-    },
+      hash: HashOp.SHA256
+    }
   };
   const tendermintSpec = {
     leafSpec: {
       prefix: Uint8Array.from([0]),
       hash: HashOp.SHA256,
       prehashValue: HashOp.SHA256,
       prehashKey: HashOp.NO_HASH,
-      length: LengthOp.VAR_PROTO,
+      length: LengthOp.VAR_PROTO
     },
     innerSpec: {
       childOrder: [0, 1],
       minPrefixLength: 1,
       maxPrefixLength: 1,
       childSize: 32,
-      hash: HashOp.SHA256,
-    },
+      hash: HashOp.SHA256
```

Comment:
Note to self: I think I exposed these in the most recent ics23 package. I should see if I can import them instead of recreating them here.